observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -743,4 +766,10 @@ The `UserWorkloadConfiguration` resource defines the settings responsible for us
 
 |thanosRuler|*link:#thanosrulerconfig[ThanosRulerConfig]|Defines the settings for the Thanos Ruler component in user workload monitoring.
 
+|namespacesWithoutLabelEnforcement|[]string|Defines the list of namespaces for which Prometheus and Thanos Ruler in user-defined monitoring do not enforce the `namespace` label value in `PrometheusRule` objects.
+
+It allows to define recording and alerting rules that can query across multiple projects (not limited to user-defined projects) instead of deploying identical `PrometheusRule` objects in each user project.
```

Comment:
```suggestion
The `namespacesWithoutLabelEnforcement` property defines recording and alerting rules that can query across multiple projects (not limited to user-defined projects) instead of deploying identical `PrometheusRule` objects in each user project.
```